src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -3151,7 +3156,19 @@ public bool DispatchKeyEvents
             }
         }
 
-
+        /// <summary>
+        /// Called when the view is hit through TouchEvent or GestureEvent.
+        /// HitTestResult is only called when touch down.
+        /// At this time, the determined hit is the same for other touch types.
+        /// If it returns true, it means that it was hit, and the touch/gesture event is called from the view.
+        /// If it returns false, it means that it will not be hit, and the hit-test continues to the next view.
+        /// User can override whether hit or not in HitTestResult.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected virtual bool HitTestResult(Touch touch)
```

Comment:
Hit 테스트할 좌표를 어떻게 구할 수 있나요? 
그 좌표는 View Bound를 기준으로 시작하는 좌표인가요, 아니면 Window screen을 기준으로 시작하는 좌표인가요?

Touch 객체가 Raw래밸의 데이터를 담고는 있지만, 실제 사용자가 필요한 정보에 바로 접근이 안되어서 차라리 `Position`과 같은 데이터로 View내의 좌표로 변환해서 argument를 주면 더 좋을것 같습니다.

Method이름도 끝에 Result가 붙어야 이유를 잘 모르겠습니다.
```suggestion
        protected virtual bool HitTest(Position position)
```

Reply:
TouchEvent처럼 Hit되었을때 Touch 데이터 그대로 제공하면 좋을 것 같다고 생각했습니다.

touch.GetLocalPosition(0) 으로 View 기준으로 시작하는 좌표를 얻어올 수 있고
touch.GetScreenPosition(0)으로 Screen 기준으로 시작하는 좌표를 얻어올 수 있습니다.

이 외에 touch.GetPointCount()를 통해 원 핑거 터치인지 투 핑거 터치인지도 구분 할 수 있습니다.

제한적인 정보보다는 이게 더 활용하기 좋으실 것 같다고 생각했는데 사용하시는 입장에서는 오히려 더 불편하실까요?

메소드 이름은 변경하도록 하겠습니다.

Reply:
Touch 클래스에 설명이 너무 부실한것이 문제인듯합니다. argument인 point에 대한 설명이 너무 이상한것 같습니다.
```
        /// <summary>
        /// Retrieves the coordinates relative to the top-left of the hit actor at the point specified.<br />
        /// The top-left of an actor is (0.0, 0.0, 0.5).<br />
        /// If you require the local coordinates of another actor (for example, the parent of the hit actor),
        /// then you should use Actor::ScreenToLocal().<br />
        /// If a point is greater than GetPointCount(), then this method will return Vector2.Zero.<br />
        /// </summary>
        /// <param name="point">The point required.</param>
        /// <returns>The coordinates relative to the top-left of the hit actor of the point specified.</returns>
        /// <since_tizen> 3 </since_tizen>
        public Vector2 GetLocalPosition(uint point)
```

설명만 잘 추가되고 여기 API의 description에도 GetLocalPosition에 대한 설명이 추가되면 Touch객체를 전달하는것도 괜찮은것 같습니다. 또한 상황에 따라서 State 정보가 중요해질수도 있을것 같아서, 모든 정보가 포함된 객체를 전달하는것이 나을것 같습니다.

Reply:
Touch 설명은 https://github.com/Samsung/TizenFX/pull/3995 에서 업데이트 하도록 하겠습니다.

Reply:
GrabTouchAfterLeave 옵션이 true가 되면 touch down되었을때에만 hit-test가 내부적으로 이뤄집니다.
그래서 굳이 C#단에서 down 처리를 할 필요가 없었습니다.

GrabTouchAfterLeave 옵션의 기본값에 대해선 검토 예정입니다.

Reply:
GrabTouchAfterLeave가 false인 상황에서는, 모든  State(Down,Move,Up)에 대해서 hit-test가 발생할것이고, 이때는 사용자가 State값을 알 필요가 있다는 의미였습니다.
여튼, Touch 자료형태로 전달하더라도 괜찮은것 같습니다.